.github/workflows/nightly.yml

```diff
@@ -42,6 +42,13 @@ jobs:
     needs: GENERATE_MATRIX
     if: ${{ needs.GENERATE_MATRIX.outputs.branches != '[]' }}
     services:
+      mysql:
+        image: mysql:8.3
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_DATABASE: test
+          MYSQL_ROOT_PASSWORD: root
```

Comment:
This is problematic, because it will add the service to older branches in nightly, at which point the `sudo service mysql start` again needs to be removed. So maybe this needs to target 8.2 after all. Or we can skip the service somehow on older branches, but I don't think that's possible, I don't see such an option in the documentation.

Reply:
Argh, just committed to PHP-8.3 and master (I want todays tags to run on all CI). What to do now?